app/services/katello/candlepin/consumer.rb

```diff
@@ -152,6 +152,10 @@ def self.distribution_to_puppet_os(name)
           'Debian'
         elsif name =~ /ubuntu/
           'Ubuntu'
+        elsif name =~ /oracle/
+          'Oracle'
```

Comment:
I noticed that the other PR uses 'OracleLinux' and yours uses 'Oracle', what is the proper puppet OS name?

Reply:
Looks like OracleLinux is the correct one

https://github.com/puppetlabs/facter/blob/79e5c272cea78dba2fbfac668335529630f79716/lib/inc/facter/facts/os.hpp#L55